src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -318,6 +323,14 @@ debug_configuration::debug_configuration()
         exit(0);
     }
 
+    if (verbose_file.length() > 0) {
+            static std::ofstream fout;
```

Comment:
nit: something is wrong with indentation

Reply:
fixed, thanks